crates/runc-shim/src/main.rs

```diff
@@ -27,7 +29,21 @@ mod task;
 
 use service::Service;
 
+fn parse_version() {
+    let os_args: Vec<_> = env::args_os().collect();
+    let flags = parse(&os_args[1..]).unwrap();
+    if flags.version {
+        println!("{}:", os_args[0].to_string_lossy());
+        println!("  Version: {}", env!("CARGO_PKG_VERSION"));
+        println!("  Revision: {}", env!("CARGO_GIT_HASH"));
```

Comment:
Is it possiable to add "dirty" if some codes changed but witout committed?

Reply:
I am trying to understand the purpose of this info - is it mostly for local debugging?

Reply:
yes, I think it is mostly useful when running locally, if you think you are on the wrong commit but actually have a few commits in it. 

```
git describe --tags --dirty --long  
```

 should give you that information

Reply:
```bash
containerd-shim-runc-v2:
  Version:  v1.7.3
  Revision: 7880925980b188f4c97b462f709d0db8e8962aff
  Go version: go1.20.6
```

This is what runc produces for version. How closely do we want to follow it? Obviously the Go version doesn't make sense here...

Reply:
And this is what this PR adds

```bash
./target/debug/containerd-shim-runc-v2-rs:
  Version: 0.1.1
  Revision: 60dc92b9d56a2fb5fc9b58e02b1e7929320608c0

Reply:
```bash
$ containerd --version
containerd github.com/containerd/containerd e86e562da.m e86e562dad79a422b6b5a8ec2292ffc61852cf09.m
```
containerd binary already has this abaility, printing".m" if the repo is changed and uncommitted.
In another word, if the output of `git status -s` is empty, it means that all changes are committed. If not, a "dirty" or ".m" should be append to git revision.

I can do that lately if that is not convenience for you.

Reply:
I can add `.m` to the postfix of revision

Reply:
Done.

Reply:
OK. Thanks